README.md

```diff
@@ -1,69 +1,55 @@
 [![license](https://img.shields.io/github/license/mainsail-crew/crowsnest?style=flat-square&color=green)](https://github.com/mainsail-crew/crowsnest/blob/master/LICENSE) [![language](https://img.shields.io/github/languages/top/mainsail-crew/crowsnest?style=flat-square&logo=gnubash&logoColor=white)](https://github.com/mainsail-crew/crowsnest/search?l=shell) [![commit-activity](https://img.shields.io/github/commit-activity/m/mainsail-crew/crowsnest?style=flat-square)](https://github.com/mainsail-crew/crowsnest/commits) [![issue-search](https://img.shields.io/github/issues/mainsail-crew/crowsnest?style=flat-square)](https://github.com/mainsail-crew/crowsnest/issues) [![discord](https://img.shields.io/discord/758059413700345988?color=%235865F2&label=discord&logo=discord&logoColor=white&style=flat-square)](https://discord.gg/mainsail)
 
+# Crowsnest the 'lookout point' for your 3d Printer.
 <p align="center">
   <picture>
     <source media="(prefers-color-scheme: dark)" srcset=".github/crowsnest-logo-darkmode.png">
     <source media="(prefers-color-scheme: light)" srcset=".github/crowsnest-logo-lightmode.png">
-    <img alt="Crowsnest Logo" src=".github/crowsnest-logo-lightmode.png" style="width: 30%">
+    <img alt="Crowsnest Logo" src=".github/crowsnest-logo-lightmode.png" style="width: 20%">
   </picture>
 </p>
 
-# crowsnest
+## What is "crowsnest"
 
-A webcam daemon for Raspberry Pi OS Lite images like mainsailOS
-
----
-
-## It inherited his name from Sail ships Crow's nest.
+Crowsnest is a webcam daemon for Unix-like OS (e.g. raspian, debian)
 
+## What has that to do with my 3d Printer 
+It inherited it´s name from Sail ships Crow's nest.
 > A crow's nest is a structure in the upper part of the main mast of a ship or a structure that is used as a lookout point.\
 > See https://en.wikipedia.org/wiki/Crow's_nest
 
 So, this will be the 'lookout point' for your Printer.
 
-### Install on Raspberry Pi OS
+## What software do i need ?
+Hopefully any Unix OS will do, but we tested it with Raspian and Debian 5.10
+It works with Mainsail and Fluidd
+
+### Install 
 
     cd ~
     git clone https://github.com/mainsail-crew/crowsnest.git
```

Comment:
Please take a look at https://github.com/mainsail-crew/crowsnest/blob/develop/README.md

This isnt true anymore, it misses a critical step!